contrib/platform/src/com/sun/jna/platform/win32/WinBase.java

```diff
@@ -1027,4 +1027,34 @@ public DWORD callback(ByteByReference pbData, Pointer pvCallbackContext,
         public DWORD callback(ByteByReference pbData, Pointer pvCallbackContext,
                               ULONGByReference ulLength);
     }
+    
+    int PIPE_CLIENT_END=0x00000000;
+    int PIPE_SERVER_END=0x00000001;
+
+		/* Pipe open mode values */
+	int PIPE_ACCESS_DUPLEX=0x00000003;
```

Comment:
Please cleanup the whole space/tab thing, these look misaligned. Sorry for the hassle.